src/main/java/com/miraisolutions/xlconnect/ResultWithAttributes.java

```diff
@@ -0,0 +1,44 @@
+package com.miraisolutions.xlconnect;
+
+import java.util.*;
+
+/**
+ * Represent attributes to be set on an object in R as part of the result to be returned.
+ * Should be extended for each required type. Not using a generic typed value, because it looks like R can't retrieve it
```

Comment:
I guess the comment should say "Not using a generic type **for attributes**" ...

Reply:
Right, actually this comment makes no sense anymore, didn't review it after my changes